main.py

```diff
@@ -95,22 +122,22 @@ def get_host_ips():
     return ip_list
 
 
-def log_startup_message(port_num, is_https_enabled, host):
+def log_startup_message(port_num, is_https_enabled, hostname):
```

Comment:
Why this change? Can we revert this back (to keep the PR to minimum changes)?

Reply:
The linter was suggesting something, Sorry didn't know

Reply:
reverting back sounds good